interactive_engine/executor/ir/graph_proxy/src/apis/graph/element/property.rs

```diff
@@ -280,3 +295,40 @@ impl Decode for DefaultDetails {
         Ok(DefaultDetails { inner: map })
     }
 }
+
+#[allow(dead_code)]
+#[derive(Clone, Debug, Default)]
+pub struct EmptyDetails {}
```

Comment:
This can implement `Copy`.

Reply:
I have removed EmptyDetails. Instead, use `Option<Arc<dyn Details>>` in DynDetails